### PR TITLE
Wansview_Q6

### PR DESCRIPTION
**Pull request by Choccy-vr**
Add the gpio pins for the Wansview_Q6.

### CODE REVIEW

File: environment/wansview-q6.uenv.txt [hunk 1]
```diff
@@ -3,18 +3,16 @@ day_night_min=500
 disable_eth=true
 enable_updates=true
 gpio_button=14
-gpio_default=
-gpio_ir850=49
+gpio_ir940=49
```
Comment (themactep) [inline environment/wansview-q6.uenv.txt:6]: if you can see the IR LED with a naked eye, it's 850nm, not 940

File: environment/wansview-q6.uenv.txt [hunk 2]
```diff
@@ -3,18 +3,16 @@ day_night_min=500
...
-gpio_motor_h=
-gpio_motor_v=
+gpio_motor_h=78 75 76 77
+gpio_motor_v=53 51 82 81
 gpio_speaker=63o
-gpio_wlan=
-motor_maxstep_h=
-motor_maxstep_v=
+motor_maxstep_h=4000
+motor_maxstep_v=4000
```
Comment (themactep) [inline environment/wansview-q6.uenv.txt:15]: i doubt the vertical steps are 4000. please double check the value.

File: environment/wansview-q6.uenv.txt (hunk 1 above)
Comment (Choccy-vr) [reply environment/wansview-q6.uenv.txt:6]: It is not visible with the naked eye.

**Comment by 112cxyz** [comment]
Did a build for this firmware ever get made?

**Comment by Choccy-vr** [comment]
I made one seems to work good but when using the video url it is a bit laggy

**Comment by 112cxyz** [comment]
Do you still have the image, because it refuses to build on my machine.

**Comment by Choccy-vr** [comment]
Yeah give me a sec. Once I have I will start a pull request with it

**Comment by Choccy-vr** [comment]
I realized it gets built from github actions and I don't really know how to add one with out being the repo owner so I will just paste my build. This is built just today so it has all the updates. Here is the bin files
[images.zip](https://github.com/user-attachments/files/20303591/images.zip)

**Comment by 112cxyz** [comment]
oh interesting your Q6 has a 16mb rom? mines only 8mb

**Comment by Choccy-vr** [comment]
Maybe an older model. You sure it is the Q6 and not a Q5 or another model

**Pull request merged**